<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>361</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="5">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>408</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CPU load:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="digitCount">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="7">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>468</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" rowspan="5">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>138</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Standby threshold (seconds)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" rowspan="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>78</width>
         <height>58</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="5" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>CPU threshold (percent)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7" rowspan="5">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>138</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="4" column="5" colspan="2">
      <widget class="QSpinBox" name="spinBoxCPUThreshold">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="5" colspan="2">
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="5">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>378</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1" colspan="5">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxCPUThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxCPUThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
